## 1. Juanita wears a red shirt 50% of the time. Her brother, Bob, wears a red shirt 20% of the time. Assuming they select their shirts independently, what is the probability that Juanita and Bob will both wear a red shirt on the same day? 

In [1]:
red_P_J<-0.50
red_P_B<-0.20
red_P_both<-red_P_J*red_P_B  # both are independent
paste(red_P_both*100, "%")

[1] "10 %"

# 2. Lets assume the observed probability of Bob and Juanita wearing a red shirt simultaneously is = 0.001.  What can you infer about relationship between these probabilities? 

### Since the probability here is too much less than 0.1 so can be inferred that both of them wear the same red shirt same day very rarely.There exists some dependency.

# 3.	A new screening blood test is designed to detect early HIV infection. 1000 people are screened, out of which 200 come out positive on the test. A gold standard procedure is then used to confirm whether a patient is actually infected with HIV. It turns out that only 100 people are actually infected with HIV, out of which 80 people have positive results on the screening test. What are the sensitivity and specificity of the screening test?  What are the positive predictive value and the negative predictive value of the test? 

                     yes hiv 	   no hiv	Total
        test pos	 80	           120	    200
        test neg	 20	           780	    800
           Total 	100	          900	   1000
            
            sentitivity=p(yes hiv and test pos)/p(hiv tot)
                        =80/100=0.80=80%
                        
            specificity=p(no hiv and test neg)/p(no hiv tot)
                       =780/900=0.866=86.6%
                       
            Positive predictive value=p(yes hiv and test pos)/p(pos tot)
                                     = 80/200=0.40=40%
            Negative predictive value=p(no hiv and test neg)/p(neg tot)
                                     =780/800=0.975=97.5%
           

# 4. At a middle school, 18% of all students play football and basketball and 32% of all students play football. What is the probability that a student plays basketball given that the student plays football? 

### given, P(F ∩ B ) = 18%= 0.18 and P(F)= 32%=0.32 
### P(B|F)=  P(F ∩ B )/P(F)
### = 0.18/o.32= 0.5625


# 5. Suppose that the height of American men is normally distributed, with a mean of 68 inches and standard deviation of 3 inches. Some men, in an effort to grow taller, have been taking an herbal supplement, marketed as “Spring Up.” You want to know if the supplement works as advertised. You randomly sample 25 of these men (the ones taking the supplement), measure their heights, and find that the average height for the sample is 68.5 inches. 

## a. State your scientific hypothesis.

### The hypothesis here is that the height of people taking supplement "spring up" is greater than those not taking. There is not any previously available data telling that height of supplement takers are lesser.

## b. State your statistical hypotheses. Will this be a one- or two-tailed test?

Ho: Supplement takers height = No Supplement takers height
i.e. H0: μ=68

H1: Supplement takers height != No Supplement takers height
i.e. H1: μ !=68

critical value=2.064, It will be two-tailed test.

## c. Choose the test statistic you will use to answer your question (Hint: we’ve only talked about one so far). Set your alpha level (α = .05).

Here,alpha=0.5 and critical value=2.04 and since, sample size=25 which is less than 30 t-test should be done 
instead of z-test 


## d. Compute your test statistic and make your decision about the statistical hypotheses.

In [25]:
#t-test=M-mu/(s/root n)
M=68.5
mu=68
s=3
a=sqrt(25)
t<-(M-mu)/(3/5)
t

[1] 0.8333333

In [11]:
pnorm(68,68.5,3)

[1] 0.4338162

### e. State your conclusion.

here 0.833<critical value(2.04) and p value is 0.433<0.05 so null hypothesis is rejected.

### f. Follow-up question (put your thinking caps on!): Suppose you are hired as a consultant to the firm that sells “Spring Up.” What criticism(s) might you offer about the research conducted above, if it calls into question the effectiveness of their herbal supplement in increasing men’s heights?

I would like to suggest that since there is not much difference in mean heights between population and sample as well as size of sample is also small,  just one time result may not be rigid one. It would be better if larger sample size taken for the test to come up with better result.